dangerzone/conversion/doc_to_pixels.py

```diff
@@ -146,14 +138,14 @@ async def convert(self) -> None:
                 "type": "libreoffice",
             },
             # .jpg
-            "image/jpeg": {"type": "convert"},
+            "image/jpeg": {"type": None},
```

Comment:
Let's not forget to create an issue for the extra file formats that PyMuPDF supports, and adding those here.